### ISSUE TITLE
Code Group not working in production

### ISSUE THREAD

**Issue by YesidMarin** [nuxt-themes/docus#975]
Hello,

I have this component in local but when I release it in prod it doesn't work. I can't copy the code and it's not formatted.

<img width="721" alt="image" src="https://github.com/nuxt-themes/docus/assets/29234631/af8a7e4c-8408-4abd-9e0c-7800df7decba">
<img width="693" alt="Screenshot 2023-09-03 at 21 44 07" src="https://github.com/nuxt-themes/docus/assets/29234631/06f9f632-7019-43c9-91c3-c59404e3e602">

**Comment by MangMax** [comment]
I seem to have the same problem and it doesn't work in the development environment

**Comment by collinped** [comment]
Happened to me as well with version v1.14.6, ended up rolling back a couple of versions.

**Comment by mathieunicolas** [comment]
Yup, it happens to me as well. It doesn't work in dev and in prod.

**Comment by collinped** [comment]
@mathieunicolas I rolled back to 1.14.3

**Comment by mathieunicolas** [comment]
> @mathieunicolas I rolled back to 1.14.3

Thanks for your answer. I tried with 1.14.3 and I still have the bad behaviour, I don't understand where it comes from. 

```
  "devDependencies": {
    "@nuxt-themes/docus": "1.14.3",
    "@nuxt/devtools": "^0.8.3",
    "nuxt": "^3.7.1",
    "rehype-katex": "^6.0.3",
    "remark-math": "^5.1.1"
  },
```

I deleted the `node_modules` and `.nuxt` folders, and... same problem in all my code-group components.
What's wrong ? My `nuxt` version ?

How can I fix that ?

**Comment by Barbapapazes** [comment]
seems related to https://www.npmjs.com/package/@nuxt-themes/elements but the repo is not opened. 🥲

**Comment by mathieunicolas** [comment]
`alert` components are broken too ? below the output for the following code :

``` 
::alert{type="info"}
Lorsque l'on affecte une valeur à une variable, il faut toujours penser au type que l'on va créer.

Par exemple, l'expression `var = 42` va créer un **entier** égal à 42. En revanche, l'expression `var = "42"` va créer une **chaîne de caractères** composée des caractères `4` et `2`.

Les fonctions `int()` et `str()` permettent de modifier le type d'une variable.
::
``` 

![image](https://github.com/nuxt-themes/docus/assets/65437872/b21f4bba-1a89-4b11-90a6-d729247204c8)

@Atinux do you have any clue for this bug ?

**Comment by Atinux** [comment]
I am sorry about this, we are in the migration to remove all sub dependencies in order to migrate to pure TailwindCSS.

I know @bdrtsky is working on it, what is the ETA?

**Comment by mathieunicolas** [comment]
> I am sorry about this, we are in the migration to remove all sub dependencies in order to migrate to pure TailwindCSS.
> 
> I know @bdrtsky is working on it, what is the ETA?

Thanks for answering, and don't be sorry !! You're doing an amazing work and huge thanks for that !
Is moving docus to tailwind in the list as well ? I was starting to think a docus remake with tailwind could be useful, and then I've seen things for Docus v2, is there a milestone somewhere if we want to help ?
